webpack.config.babel.js

```diff
@@ -0,0 +1,140 @@
+const path = require('path')
+const webpack = require('webpack')
+const cssnano = require('cssnano')
+const CopyWebpackPlugin = require('copy-webpack-plugin')
+const HtmlWebpackPlugin = require('html-webpack-plugin')
+const WriteFilePlugin = require('write-file-webpack-plugin')
+
```

Comment:
You are using ".babel.js".
Why use "require"?

For example "import path from 'path';" is not it?